$OpenBSD: patch-test_git_test_py,v 1.1 2018/02/18 11:35:48 bket Exp $

test_tag and test_show assume that source has been fetched using git, and that
full history is availabe. These tests fail as we are using a release tarball.

Adapted from
https://github.com/git-cola/git-cola/commit/4c9d36ae021262a6559a1ae240c31e768bca0b37

Index: test/git_test.py
--- test/git_test.py.orig
+++ test/git_test.py
@@ -219,19 +219,6 @@ class GitCommandTest(unittest.TestCase):
         version = self.git.version()[STDOUT]
         self.failUnless(version.startswith('git version'))
 
-    def test_tag(self):
-        """Test running 'git tag'"""
-        tags = self.git.tag()[STDOUT].splitlines()
-        if os.getenv('GIT_COLA_NO_HISTORY', False):
-            return
-        self.failUnless('v1.0.0' in tags)
-
-    def test_show(self):
-        """Test running 'git show'"""
-        oid = 'HEAD'
-        content = self.git.show(oid)[STDOUT]
-        self.failUnless(content.startswith('commit '))
-
     def test_stdout(self):
         """Test overflowing the stdout buffer"""
         # Write to stdout only
